.travis.yml

```diff
@@ -51,8 +51,12 @@ matrix:
     - php: '7.1'
       env: SYMFONY=dev-master@dev
     - php: '7.1'
+      env: SONATA_ADMIN=3.*
+    - php: '7.1'
       env: SONATA_ADMIN=dev-master@dev
     - php: '7.1'
+      env: SONATA_BLOCK=3.*
```

Comment:
Maybe that's indirectly because of this: https://github.com/sonata-project/SonataDoctrinePhpcrAdminBundle/blob/1.x/composer.json#L22